android/common/src/main/java/org/kotlinacademy/common/ContextExt.kt

```diff
@@ -11,18 +11,17 @@ val Context.notificationManager: NotificationManager
 fun Context.openUrl(url: String?) {
     url.nullIfBlank() ?: return
     val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
-    if (browserIntent.resolveActivity(packageManager) != null) {
```

Comment:
I intentionally left it this way to keep it easier and more intuitive (especially for people from outside Kotlin world)

Reply:
Ok--will revert this change